tools/epicseg/.shed.yml

```diff
@@ -0,0 +1,14 @@
+name: epicseg
+owner: wkopp
```

Comment:
We do have a team called `rnateam` to we share the burdon of maintaining tools in de.NBI.
This owner will become a TS owner, so if you like but rnateam here :)